spec/models/project_spec.rb

```diff
@@ -98,6 +98,20 @@
     end
   end
 
+  describe "#active_workflows" do
+    let(:project) do
+      create(:project) do |p|
+        create(:workflow, project: p, active: true)
+        create(:workflow, project: p, active: false)
+      end
+    end
+
+    it "should have only one active workflow" do
```

Comment:
I think that projects can have more than one active workflow. Seems that it's just this single spec that seems to disagree.